### ISSUE TITLE
We need labeling of emojis

### ISSUE THREAD

**Issue by pomkos** [pillargg/pillar_algos#25]
I want to start expanding `algo3_5.py`. Right now it can count words/emojis/both per person. I would like to expand it to count "sentiments" per person.

Good way to start doing this is by labeling the `emoticons` that Twitch uses. They have [definitions](https://www.twitch.tv/creatorcamp/en/learn-the-basics/emotes/) but these definitions need to become more categories. Maybe 6 categories (fear, anger, joy, sadness, surprise and disgust). These follow the "basic emotions" theory, there was a [recent meta-analysis](https://www.frontiersin.org/articles/10.3389/fpsyg.2017.01432/full#h9) of this concept and also here is a [wiki](https://en.wikipedia.org/wiki/Emotion_classification#Emotions_as_discrete_categories).

Anyways, 6 categories for the emoticons doable? Whoever knows Twitch culture should take the lead on this.

I added Matt and Steve cuz I know they do Twitch a lot, but feel free to assign yourselves!

**Comment by pomkos** [comment]
Commit d503437 includes the [`data/emoticons.csv`](https://github.com/pillargg/twitch_chat_analysis/blob/unit_testing/data/emoticons.csv), scraped from Twitch link.

**Comment by pomkos** [comment]
@gatesyp and I talked. We might ask users to label their channel's most frequently used custom emoticons

Default twitch emoticons: label them as they come up, create a table in csv. @gatesyp @Geczy  will label them.

**Comment by gatesyp** [comment]
Quick note to expand on the labelling tool for users -- thats a nice way to scale the categorization of custom emotes. Not needed now, but if this approach yields good results then we can consider building the labelling tool for users.

**Comment by pomkos** [comment]
We now have the ability to extract all unique emoticon ids and names, along with the number of times they were used in a particular dataset. [script here](https://github.com/pillargg/twitch_chat_analysis/blob/unit_testing/pypi/prod/pillaralgos/helpers/emoji_getter.py)

Output [here](https://github.com/pillargg/twitch_chat_analysis/blob/unit_testing/data/top_emoticons.csv) in csv form.

**Comment by pomkos** [comment]
Merged into unit_testing branch, cuz thats where all my deving is happening.

**Comment by Geczy** [comment]
amazon has a labeling tool for training data sets

https://aws.amazon.com/getting-started/hands-on/build-training-datasets-amazon-sagemaker-ground-truth/

![image](https://user-images.githubusercontent.com/1036968/114872768-9ee8a980-9dc8-11eb-90e5-de7b3ad36fc7.png)

**Comment by pomkos** [comment]
We need more labeled emoticons @gatesyp @Geczy
